TypeScriptReact.YAML-tmLanguage

```diff
@@ -0,0 +1,720 @@
+# Similar to TypeScript.YAML-tmLanguage except:
+#   - Remove type assertion (cast.expr.ts)
```

Comment:
If this is largely a copy, did you consider having the source.tsx language just include the source.ts language? See https://www.sublimetext.com/forum/viewtopic.php?f=6&t=16560#p62430